UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/Helper/EnsApiClient.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using UKHO.ExternalNotificationService.API.FunctionalTests.Model;
+
+namespace UKHO.ExternalNotificationService.API.FunctionalTests.Helper
+{
+    public class EnsApiClient
+    {
+        static readonly HttpClient s_httpClient = new();
```

Comment:
Generally in .netcore it is better to use the IHttpClientFactroy

Reply:
Thanks for highlighting this, we are working on it, but it takes time.
Please don't hold this for now, we will cover the changes in the upcoming pr.